apps/condo/domains/user/schema/User.test.js

```diff
@@ -31,8 +31,8 @@ describe('SIGNIN', () => {
             'identity': DEFAULT_TEST_USER_IDENTITY,
             'secret': DEFAULT_TEST_USER_SECRET,
         })
-        expect(data.auth.user.id).toMatch(/[a-zA-Z0-9-_]+/)
         expect(errors).toEqual(undefined)
+        expect(data.obj.item.id).toMatch(/[a-zA-Z0-9-_]+/)
```

Comment:
better to move static regexps to constants outside the function

Reply:
it's useless for reuse. It's ok to use such things inside the tests